### ISSUE TITLE
Brave is detectable

### ISSUE THREAD

**Issue by AhsanE** [brave/browser-laptop#11012]
Brave can be detected. There are 7 possible ways to easily identify brave, without canvas and shields off. Will be disclosed on hacker one, if bounty eligible. Else solution will be released commercially,

**Comment by luixxiul** [comment]
cc @diracdeltas

**Comment by luixxiul** [comment]
@diracdeltas will this be closed in favor of https://github.com/brave/browser-laptop/issues/10655?

**Comment by AhsanE** [comment]
@luixxiul At that time there were 3 ways. Now I've found 7 ways. Even with the latest update there were no fixes.

**Comment by diracdeltas** [comment]
Although our HackerOne bounty program lists detecting Brave as out-of-scope, I would accept this as eligible if you found an attack that sets the User-Agent (or some other commonly-logged browser property) to Brave instead of Chrome OR if the attack worked with scripts disabled.

**Comment by AhsanE** [comment]
The attack can detect without JavaScript enabled.

**Comment by luixxiul** [comment]
@diracdeltas please close this if this will be also triaged through hackerone.

**Comment by AhsanE** [comment]
@diracdeltas Will wait for 1 business day for the reply. Otherwise response will be assumed as negative.

**Comment by diracdeltas** [comment]
closing this and waiting on the hackerone report, thanks
